src/components/Auth.js

```diff
@@ -0,0 +1,87 @@
+import auth0 from 'auth0-js';
+
+let callbackURL;
+
+if(process.env.NODE_ENV === 'development'){
+    callbackURL = `http://localhost:3000/callback`
+} else if (process.NODE_ENV === 'production' || process.NODE_ENV !== 'development'){
+    callbackURL = `https://www.wellbroomed.com/callback`
+    // netlify callback URL goes here
+}
+
+class Auth {
+  constructor() {
+    this.auth0 = new auth0.WebAuth({
```

Comment:
Ran into `Error: domain option is required` locally on the browser. Not sure what this is about. I also don't have the dotenv file so that could be it.

No errors in my terminal though other than unused styled and imports.

Reply:
.env values posted in Slack 👍